admin/create-dumps.sh

```diff
@@ -136,12 +136,7 @@ DUMP_NAME=$(basename "$DUMP_DIR")
 # Backup dumps to the backup volume
 # Create backup directories owned by user "listenbrainz"
```

Comment:
Also, this comment looks wrong. We don't have a `listenbrainz` user. Should I remove it?

Reply:
yes

Reply:
looking at this more, [we _do_ have a listenbrainz user](https://github.com/metabrainz/listenbrainz-server/blob/d540cdbabe2fcd1fae9d84b105fe2373c039a177/Dockerfile#L53), though we don't use it for dumps, and this comment is out of date, so it needs to be fixed anyway